server/apps/publish/content/correct.py

```diff
@@ -26,6 +26,7 @@ def on_update(self, updates, original):
         ArchiveCropService().validate_multiple_crops(updates, original)
         super().on_update(updates, original)
         updates[ITEM_OPERATION] = ITEM_CORRECT
+        updates['versioncreated'] = utcnow()
```

Comment:
Any specific reason for not updating versioncreated on publish? I think there's no difference between publish, correct and kill operations.